CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### 1.6.0 2/5/2016
+* Removed the contact form captcha requirement for logged in users.
+* Added the ability to assign read-only permissions to a group.
```

Comment:
Should read: `Added the ability to assign read-only permissions to individuals or to a group.`